fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -41,7 +41,10 @@ object ItemKicker {
     }
   }
 
-  def fromTrailMetaData(trailMeta: MetaDataCommonFields): Option[ItemKicker] = fromContentAndTrail(None, trailMeta, ResolvedMetaData.Default, None)
+  def fromTrailMetaData(trailMeta: MetaDataCommonFields): Option[ItemKicker] = fromContentAndTrail(None, trailMeta, ResolvedMetaData.fromTrailMetaData(trailMeta), None)
```

Comment:
Of course they could do that. I think that makes the code a lot messier. And this is a common enough format to not be unfamiliar.